### DISCUSSION TITLE
最新版本，还是有ssl 访问有问题，在不导入证书时，可以访问页面，如果导入证书后，访问就会提示ssl握手失败

### DISCUSSION THREAD

**Discussion by gjf026** [reqable/reqable-app#805] [Q&A]
最新版本，还是有ssl 访问有问题，在不导入证书时，可以访问页面，如果导入证书后，访问就会提示ssl握手失败

**Comment by MegatronKing** [comment]
证书问题和reqable没关系，这个需要自行解决的。

**Comment by gjf026** [comment]
导入的是reqable的证书。不是网站的证书

**Reply by MegatronKing** [reply]
一样的，就像网站证书和你服务器代码逻辑无关，一般都是配置不对导致的，需要你自行排错。如果你持有Reqable年度或永久许可证，可以在我们官网添加微信寻求技术支持。
